engines/ic3base.h

```diff
@@ -273,6 +252,32 @@ class IC3Base : public Prover
    */
   virtual IC3Formula ic3formula_negate(const IC3Formula & u) const;
 
+  /** Attempt to generalize before blocking in frame i
```

Comment:
is something changed here? or is it just moved?

Reply:
Just moved!